examples/workflows/model_nudging.py

```diff
@@ -0,0 +1,105 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# %%
+import os
+from functools import partial
+import torch
+import datetime
+from modulus.distributed.manager import DistributedManager
+from earth2mip.inference_ensemble import run_basic_inference
+from earth2mip.networks import get_model
+from earth2mip.initial_conditions import cds
+
+
+def apply_gaussian_perturbation(
+    x,
+    time_step,
+    in_channel_names,
+    device,
```

Comment:
remove this. get `device` from `x` or use `torch.zeros_like` `torch.ones_like`.

I believe I've made similar comments in the past. no function like this should require `device`.